tox.ini

```
@@ -17,22 +18,38 @@ commands =
     pytest -vv
 
 [testenv:flake8]
+skip_install = True
 deps =
     flake8
 commands = flake8 {posargs}
 
-[testenv:docs]
+[testenv:check-manifest]
+skip_install = True
 deps =
-    -r{toxinidir}/docs/requirements_docs.txt
+   check_manifest
 commands =
-    python setup.py build_sphinx -W
+   check-manifest -v
+
+[testenv:towncrier-check]
+skip_install = True
+deps =
+    towncrier >= 18.5.0
```

Might be a good idea to use

```suggestion
    {[testenv:towncrier]deps}
```

to ensure that both envs use the same Towncrier dependency.

---

Oh - nice. Will do.